package.json

```diff
@@ -98,7 +99,7 @@
     "lodash": "^4.15.0",
     "moment": "^2.14.1",
     "request": "^2.79.0",
-    "share2nightscout-bridge": "^0.1.5",
+    "share2nightscout-bridge": "git://github.com/nightscout/share2nightscout-bridge.git#wip/generalize",
```

Comment:
What does this do?

Reply:
This is the format Nightscout uses. We should re publish a new NPM package. I tried contacting Ben West, but didn't get a response.
For a list of changes see https://github.com/nightscout/share2nightscout-bridge/commits/wip/generalize
It fixes a security issue.